ops/global/main.tf

```diff
@@ -31,6 +31,12 @@ resource "azurerm_log_analytics_workspace" "sr" {
   tags = local.management_tags
 }
 
+// Okta preview configuration
+# module "okta_preview" {
+#   source = "../services/okta-global"
+
+#   all_users_group_id = data.okta_group.everyone.id
+# }
```

Comment:
does this comment need to be cleaned up?

Reply:
This is more of a reminder for future Alis or others to consider importing or creating oktapreview resources with terraform. I'll create a ticket to act as a reminder and remove this comment. :thought_balloon:

Reply:
any update on this action? removing and comment and creating the ticket?

Reply:
Oops! I thought I had done this. As you suggest in your approval comment, I'll get this done in a follow-up PR.